src/App.tsx

```diff
@@ -15,71 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGoods: string | null;
```

Comment:
This is 100% correct, jus wanted to point out that it also could be
`selectedGoods ?: string,`

Reply:
@jacekkosiba not quite the same.
Keep in mind that `selectedGoods?: string` is equal to `selectedGoods: string | undefined`, not `selectedGoods: string | null`